rg_instructor_analytics/static/rg_instructor_analytics/js/ProblemTab.js

```diff
@@ -47,21 +47,23 @@ function ProblemTab(button, content) {
                 index++;
             }
             bars = `<ul class="plot-body">${bars}</ul>`;
-            //build y axis
+            //build x axis
             let axis = ``;
+            let small;
+            if (yAxis.length > 10) {small = 'small'}
             yAxis.forEach((item) => {
-                axis += `<li style="width: ${(100 - yAxis.length) / yAxis.length}%;"><div>${item}</div></li>`
+                axis += `<li class="hw-xaxis ${small}" style="min-width: ${(100 - yAxis.length) / yAxis.length}%;"><div>${item}</div></li>`
```

Comment:
Could you please split this line into two sp review will be much easier.

Reply:
done